src/TestFramework/DataFaker.php

```diff
@@ -1160,13 +1160,24 @@ public function path()
     }
 
     /**
-     * The token received from Apple Pay payment sheet.
-     * Includes the country, zip code, expiration date and account holder name.
-     * Need to use json encode so that it is parsed as a string instead of an array to match token object.
+     * Returns a card token
      *
      * @return string
      */
     public function token()
+    {
+        return $this->randomCharacters(self::DIGITS . self::ALPHABET_UPPER, 17);
+    }
+
+    /**
+     * The paymentData portion of the token taken from the ApplePayPayment object received from
+     * the Apple Pay payment sheet.
+     * We need to use json encode it so that its parsed as a string instead of an array to
+     *  match token object.
+     *
```

Comment:
A few typos here .

> The paymentData portion of the token taken from the ApplePayPayment object received from
> \* the Apple Pay payment sheet.

Can you clarify that this is what's returned.

> We need to use json encode it so that its parsed as a string instead of an array to
> \*  match token object.

* Extra space before `match`
* its should be it's
* `use` can be omitted.